test/spec/modules/taboolaBidAdapter_spec.js

```diff
@@ -386,6 +387,31 @@ describe('Taboola Adapter', function () {
       const res = spec.interpretResponse(serverResponse, request)
       expect(res).to.deep.equal(expectedRes)
     });
+
+    it('should set the correct ttl form the response', function () {
+      // set exp-ttl to be 125
+      const [bid] = serverResponse.body.seatbid[0].bid;
+      serverResponse.body.seatbid[0].bid[0].exp = 125
```

Comment:
semicolon?

Reply:
fixed - added semicolon

Reply:
in the lines below as well, our lgtm ends up flagging missing semicolons

Reply:
tyvm, please note the trailing spaces causing the failed linting checks